src/RegistryCoordinator.sol

```diff
@@ -298,7 +298,7 @@ contract RegistryCoordinator is
         address[][] calldata operatorsPerQuorum,
         bytes calldata quorumNumbers
     ) external onlyWhenNotPaused(PAUSED_UPDATE_OPERATOR) {
-        uint192 quorumBitmap = uint192(BitmapUtils.orderedBytesArrayToBitmap(quorumNumbers));
+        uint192 quorumBitmap = uint192(BitmapUtils.orderedBytesArrayToBitmap(quorumNumbers, quorumCount));
```

Comment:
seems odd that this change slipped into this PR?

Reply:
oops, probably just reflexively made a change while reviewing for docs info.

i'm gonna keep it in though :smiling_imp: